waagent

```diff
@@ -5909,8 +5916,14 @@ def main():
         else:
             args.append(a)
     global Config
-    Config = ConfigurationProvider()
+    Config = ConfigurationProvider(conf_file)
     
+    logfile = Config.get("Logs.File")
+    if logfile is not None:
+        myLogger.file_path = logfile
+    logconsole = Config.get("Logs.Console")
+    if logconsole is not None and logconsole.lower().startswith("y"):
```

Comment:
Hi, @imikushin I was talking about upgrade scenario. When existing user upgrade to new version, I can't overwrite existing config. That's the reason I need to keep the console log as default setting.